.github/workflows/api-release.yml

```
@@ -0,0 +1,38 @@
+runs-on: ubuntu-latest
+steps:
+  - name: Check out Git repository
+    uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744 # v3
+    with:
+      ref: ${{ github.event.inputs.branch }}
+  - name: Setup git user
+    uses: fregante/setup-git-user@024bc0b8e177d7e77203b48dab6fb45666854b35 # v2.0.2
+  - name: Set up JDK 17 and OSSRH auth / GPG signing
+    uses: actions/setup-java@0ab4596768b603586c0de567f2430c30f5b0d2b0 # v3
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)